### DISCUSSION TITLE
Mp3 in psram

### DISCUSSION THREAD

**Discussion by luke2023** [pschatzmann/arduino-audio-tools#459] [Q&A] [answered]
Hello, I am making a open source watch which can store and play mp3.There are two ways i can think of to store mp3.One is store Mp3 in sd card which is possible
,but the sd module is too big. Another is to store it in a 1GB psram.But is it possible to flash mp3 from phone with http to psram and play it?Thank you!

**Comment by pschatzmann** [comment, marked as answer]
I am not aware of any API which would allow this. You can however use your watch as Bluetooth Speaker with the A2DP libary if you want to send music from the phone...

You could also try to use an SD card directly w/o module to save space.
